packages/enzyme-test-suite/test/selector-spec.jsx

```diff
@@ -430,6 +430,56 @@ describe('selectors', () => {
         expectAttributeMatch(<div data-foo={NaN} />, '[data-foo=-Infinity]', false);
         expectAttributeMatch(<div data-foo={0} />, '[data-foo=Infinity]', false);
       });
+
+      it('whitespace list attribute selector', () => {
+        expectAttributeMatch(<div rel="foo bar baz" />, '[rel~="bar"]', true);
+        expectAttributeMatch(<div rel="foo bar baz" />, '[rel~="baz"]', true);
+        expectAttributeMatch(<div rel="foo bar baz" />, '[rel~="foo"]', true);
+        expectAttributeMatch(<div rel="foo bar baz" />, '[rel~="foo bar"]', false);
+        expectAttributeMatch(<div rel={1} />, '[rel~=1]', false);
```

Comment:
Correct me if I'm wrong - shouldn't this return true? I think `~=` should still match if it's exactly equal, with no whitespace.

Reply:
@jack-lewin this implementation only supports matching strings with these partial match operators, so all other types will return `false`. If you want to match exactly equal values you should use `=`